### ISSUE TITLE
Missing Pods xcodeproject inside the node_modules folder

### ISSUE THREAD

**Issue by anandwahed** [prscX/react-native-fluidic-slider#8]
## Error

- We are facing an issue where the `Pods.xcodeproject` is missing inside the ios folder. We tried to follow the steps mentioned for the `iOS` and are facing with the following error.

```
➜  **ios git:(develop) ✗ pod install**
Analyzing dependencies
Pre-downloading: `fluid-slider` from `https://github.com/prscX/fluid-slider`, branch `objectivec`
Downloading dependencies
Installing fluid-slider (1.0.0)
Installing pop (1.0.12)
[!] Unable to determine Swift version for the following pods:

- `fluid-slider` does not specify a Swift version and none of the targets (`RNFluidicSlider`) integrating it have the `SWIFT_VERSION` attribute set. Please contact the author or set the `SWIFT_VERSION` attribute in at least one of the targets that integrate this pod.

[!] The Podfile contains framework or static library targets (RNFluidicSlider), for which the Podfile does not contain host targets (targets which embed the framework).
If this project is for doing framework development, you can ignore this message. Otherwise, add a target to the Podfile that embeds these frameworks to make this message go away (e.g. a test target).

➜  **ios git:(develop) ✗ pod repo update**
Updating spec repo `master`
  $ /usr/local/bin/git -C /Users/anand/.cocoapods/repos/master fetch origin --progress
  remote: Enumerating objects: 41, done.
  remote: Counting objects: 100% (41/41), done.
  remote: Compressing objects: 100% (14/14), done.
  remote: Total 27 (delta 18), reused 21 (delta 12), pack-reused 0
  From https://github.com/CocoaPods/Specs
     3c2d65ba61f..461115c023e  master     -> origin/master
  $ /usr/local/bin/git -C /Users/anand/.cocoapods/repos/master rev-parse --abbrev-ref HEAD
  master
  $ /usr/local/bin/git -C /Users/anand/.cocoapods/repos/master reset --hard origin/master
  HEAD is now at 461115c023e [Add] KerfuSDK 0.0.1.24

➜  **ios git:(develop) ✗ pod install**
Analyzing dependencies
Pre-downloading: `fluid-slider` from `https://github.com/prscX/fluid-slider`, branch `objectivec`
Downloading dependencies
Installing fluid-slider (1.0.0)
Installing pop (1.0.12)
[!] Unable to determine Swift version for the following pods:

- `fluid-slider` does not specify a Swift version and none of the targets (`RNFluidicSlider`) integrating it have the `SWIFT_VERSION` attribute set. Please contact the author or set the `SWIFT_VERSION` attribute in at least one of the targets that integrate this pod.

[!] The Podfile contains framework or static library targets (RNFluidicSlider), for which the Podfile does not contain host targets (targets which embed the framework).
If this project is for doing framework development, you can ignore this message. Otherwise, add a target to the Podfile that embeds these frameworks to make this message go away (e.g. a test target).

➜  **ios git:(develop) ✗**
```

## ReactNative Info

- This causes build errors while trying to run the iOS project with ReactNative. Below is the output of `react-native info` if it would be of any help.

```
  React Native Environment Info:
    System:
      OS: macOS 10.14.3
      CPU: x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
      Memory: 3.78 GB / 32.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 11.10.0 - /usr/local/bin/node
      Yarn: 1.13.0 - /usr/local/bin/yarn
      npm: 6.8.0 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.1, macOS 10.14, tvOS 12.1, watchOS 5.1
      Android SDK:
        Build Tools: 26.0.0, 26.0.3, 27.0.3, 28.0.2, 28.0.3
        API Levels: 25, 26, 27, 28
    IDEs:
      Xcode: 10.1/10B61 - /usr/bin/xcodebuild
    npmPackages:
      @types/react: 16.8.1 => 16.8.1
      @types/react-native: 0.57.32 => 0.57.32
      react: 16.7.0 => 16.7.0
      react-native: 0.58.3 => 0.58.3
    npmGlobalPackages:
      react-native-cli: 2.0.1
      react-native-create-bridge: 2.0.1
      react-native-create-library: 3.1.2
      react-native-swift-cli: 2.3.0
```

**Comment by anandwahed** [comment]
@prscX can you take a look into this issue ?

**Comment by anandwahed** [comment]
Hey @prscX did you have time to look into the above mentioned issue ?

**Comment by anandwahed** [comment]
Closing the issue since moved to a different implementation.

**Comment by ustincameron** [comment]
@anandwahed Did you go with a different slider? @prscX I have this issue as well, can you update readme instructions. Pods.xcodeproject does not exist and it complains about swift version being unknown.

**Comment by ejkkan** [comment]
+1 would like to have installation docs updated

**Comment by davidyong0418** [comment]
you can fix it with cocoapods version 1.5.3
